outputs.tf

```diff
@@ -55,10 +55,10 @@ output "ebs_ids" {
 
 output "primary_network_interface_id" {
   description = "ID of the instance's primary network interface"
-  value       = "${aws_instance.default.primary_network_interface_id}"
+  value       = "${aws_instance.default.*.primary_network_interface_id}"
 }
 
 output "network_interface_id" {
   description = "ID of the network interface that was created with the instance"
-  value       = "${aws_instance.default.network_interface_id}"
+  value       = "${aws_instance.default.*.network_interface_id}"
```

Comment:
since `${aws_instance.default.*.network_interface_id}` returns a list,
it should be `${join("", aws_instance.default.*.network_interface_id)}`
to return one item from the list

Reply:
interfaces can be more than one

Reply:
the output returns one ID, not a list (as the description says).
If returns more than one, it should be changed to

```hcl
output "network_interface_ids"
```